### ISSUE TITLE
Differences between name and offset properties

### ISSUE THREAD

**Issue by hiramdeveloper** [dmfilipenko/timezones.json#38]
For example in the (UTC-06:00) Guadalajara, Mexico City, Monterrey case, those properties has differents values, I mean has the next values below:

name: (UTC-06:00) Guadalajara, Mexico City, Monterrey
offset: -5

The correct offset property value should be: -6 like we can see in the name property.

Could you help adding the correct value please?

**Comment by CodeWorldIndustries** [comment]
Yep! I thought I was going crazy.... spent 3 days figuring out why I was always off by one hour when I had a use case between calculating delta between Pacific Standard Time and Central Standard Time... He has the offset for CST set to -5 but it's supposed to be -6 and PST is correct at -8...

@dmfilipenko I recommend you go write a script check if the text value contains the offset number.. Cheers...

**Comment by martin-sweeny** [comment]
This is because @dmfilipenko didn't take daylight savings into account, it seems. It looks like this is a snapshot of time zones at some arbitrary point in time.

**Comment by lzilioli** [comment]
EST has the same issue. I think it is cleaner to parse the offset out of the `text` property rather than rely on the numerical `offset` property provided in the list.

**Comment by adrielTosi** [comment]
I've notices several of these differences, where the `text` and `offset` values differ. For example these two:
```
  {
    value: 'Central Standard Time',
    abbr: 'CDT',
    offset: -5,
    isdst: true,
    text: '(UTC-06:00) Central Time (US & Canada)',
    utc: [ ... ],
  },
 {
    value: 'Eastern Standard Time',
    abbr: 'EDT',
    offset: -4,
    isdst: true,
    text: '(UTC-05:00) Eastern Time (US & Canada)',
    utc: [ ... ],
  },

```
Anyone got any insight into this?
